widgets/gui/qt_left_window.py

```diff
@@ -66,8 +66,8 @@ def __init__(self, parent):
 
     @property
     def update_parameters(self):
```

Comment:
this is not a method but a property, naming it like `latest_parameters_selected` can be better

Reply:
Changed!